src/src/AppBundle/Controller/DefaultController.php

```diff
@@ -3,17 +3,20 @@
 namespace AppBundle\Controller;
 
 use AdminBundle\Entity\EventType;
+use AppBundle\Entity\Attendee;
+use AppBundle\Form\RegistrationForm;
+use EmailBundle\Controller\EmailController;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
-use TemplateBundle\Controller\CustomTemplateController;
+use Symfony\Component\HttpFoundation\Request;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
 
-class DefaultController extends CustomTemplateController
+class DefaultController extends EmailController
 {
     /**
      * @Route("/{slug}/{_locale}", name="frontend_index", defaults={"_locale": "DEFAULT"})
-     * @Method("GET")
+     * @Method({"GET", "POST"})
      */
-    public function indexAction($slug, $_locale)
+    public function indexAction($slug, $_locale, Request $request)
```

Comment:
Pri registrovani, sa mi podarilo dostat chybu _Exception occurred while flushing email queue: Connection could not be established with host 127.0.0.1 [Connection refused #111]_ a tak mi napadlo ci by nebolo dobre ukladat do contextu aj error message v pripade ze nejaky nastane. Aby ho teda bolo mozne pri registraci vypisat. Tu mozu byt predsa aj errory typu ze je uz registrovany.

Reply:
Neviem, chyby typu, ze registracia uz existuje by sme mali riesit na urovni formulara (to nieje sucastou tejto issue, formular som tam pridal iba pre jednoduche testovanie). Vytvorim pre to pravdepodobne separatnu issue.

Reply:
Ok, moze byt